components/nspanel_lovelace/types.h

```diff
@@ -457,6 +458,11 @@ struct ha_action_type {
   static constexpr const char* open_cover = "open_cover";
   static constexpr const char* close_cover = "close_cover";
   static constexpr const char* stop_cover = "stop_cover";
+  static constexpr const char* set_cover_position = "set_cover_position";
+  static constexpr const char* open_cover_tilt = "open_cover_tilt";
+  static constexpr const char* close_cover_tilt = "close_cover_tilt";
+  static constexpr const char* stop_cover_tilt = "stop_cover_tilt";
+  static constexpr const char* set_cover_position_tilt = "set_cover_position_tilt";
```

Comment:
I think this is probably meant to be `set_cover_tilt_position` instead of `set_cover_position_tilt` as seen in the [HA cover docs](https://www.home-assistant.io/integrations/cover/#service-coverset_cover_tilt_position)

Reply:
That was a mistake, good find.